docs/import/color_function_util.js

```diff
@@ -16,33 +16,35 @@ export {
 // color function info is currently displayed.
 let globalTd;
 
-const getWarning = (missing) =>
-    'Warning: layer missing ' + missing + '. May not show up on map.';
-
 /**
  * Displays a warning in color function editor if field or color(s) are missing
- * @param {string|number|boolean} field
- * @param {string|boolean} color
+ * @param {string|number|boolean} hasField
```

Comment:
nit: why are these named like booleans but can be strings? Maybe just use !! in the callsites?